src/components/Modals/Send/views/Address.tsx

```diff
@@ -150,6 +169,31 @@ export const Address = () => {
     navigate(SendRoutes.Scan)
   }, [navigate])
 
+  const handleSelectAddressBookEntry = useCallback(
+    (entryAddress: string) => {
+      setValue(SendFormFields.Input, entryAddress, { shouldValidate: true })
+      navigate(SendRoutes.AmountDetails)
+    },
+    [setValue, navigate],
+  )
+
+  const handleSaveContact = useCallback(
```

Comment:
suggestion: from a product perspective, this is the only place you see "Contact" vernacular - shouldn't code/copies here use address book vernacular too?